nextflow.config

```diff
@@ -194,6 +194,9 @@ params {
     validationShowHiddenParams       = false
     validate_params                  = true
 
+    // Generate downstream samplesheets
+    generate_downstream_samplesheets = false
+    generate_pipeline_samplesheets   = "funcscan,taxprofiler"
```

Comment:
Should this default to null so that users have to opt-in to samplesheet generation?

Reply:
Ah maybe I misunderstood Carson... not sure :D